### PR TITLE
Add package for tinyformat

### PR DESCRIPTION
**Pull request by tunabrain**

### CODE REVIEW

File: mingw-w64-tinyformat/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Maintainer: Benedikt Bitterli <benedikt.bitterli@gmail.com>
+
+_realname=tinyformat
```
Comment (mingwandroid) [inline mingw-w64-tinyformat/PKGBUILD]: Please add pkgbase

Comment (tunabrain) [inline mingw-w64-tinyformat/PKGBUILD]: Ok, done

File: mingw-w64-tinyformat/PKGBUILD [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+pkgdesc="A minimal type safe printf() replacement (mingw-w64)"
+arch=('any')
+license=('boost')
+url=('https://github.com/c42f/tinyformat')
+source=("http://github.com/c42f/tinyformat/archive/Release-${pkgver}.tar.gz")
+md5sums=('b1c1e6206e4152ae021813ac40a0e015')
+
+package() {
+  cd "${srcdir}/${_realname}-Release-${pkgver}"
+  install -Dm644 tinyformat.h "${pkgdir}/${MINGW_PREFIX}/include/tinyformat.h"
```
Comment (Alexpux) [inline mingw-w64-tinyformat/PKGBUILD]: Remove slash between ${pkgdir} and ${MINGW_PREFIX}

Comment (tunabrain) [inline mingw-w64-tinyformat/PKGBUILD]: Ok, done.

The path was modeled after the [ladspa PKGBUILD](https://github.com/Alexpux/MINGW-packages/blob/master/mingw-w64-ladspa-sdk/PKGBUILD), would that need to be fixed as well?

**Comment by Alexpux** [comment]
Yeah fix it in ladspsa too

**Comment by tunabrain** [comment]
Oh, it looks like ladspa.org is dead, and I can't even build the package.

**Pull request merged**